src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -194,7 +194,7 @@ protected function file($file, $fileName = null, $disposition = ResponseHeaderBa
     protected function addFlash($type, $message)
     {
         if (!$this->container->has('session')) {
-            throw new \LogicException('You can not use the addFlash method if sessions are disabled.');
+            throw new \LogicException('You can not use the addFlash method if sessions are disabled. Enable them in "config/packages/framework.yaml"');
```

Comment:
Error messages like this -> *Enable them in "config/packages/framework.yaml"* could be problematic for 3.4 app ... which probably use the same dir layout that 3.3. The new layout is only generally used in 4.0 and newer.

Reply:
Most of the exceptions will probably never be shown if you depend on the Standard Edition (this one is indeed a bit different, but we could additionally check for the `FullStack` class).